### DISCUSSION TITLE
`test-bot` workflow fails to build Rust formula with the error `failed to verify the checksum of `miniz_oxide v0.5.1``

### DISCUSSION THREAD

**Discussion by Southpaw1496** [Homebrew/discussions#4223] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I'm trying to add a formula to a custom tap that I made with `brew new-tap` (it's beta software so I didn't want to put it in core). When I opened a pull request, the `test-bot` workflow failed when trying to run `brew install --verbose --build-bottle southpaw1496/quilt/quilt-native-installer` with the following error:

```
error: failed to compile `quilt-installer v0.1.1 (/tmp/quilt-native-installer-20230216-12213-1yxesg/quilt-native-installer-0.1.1)`, intermediate artifacts can be found at `/tmp/quilt-native-installer-20230216-12213-1yxesg/quilt-native-installer-0.1.1/target`
[58](https://github.com/Southpaw1496/homebrew-quilt/actions/runs/4195113451/jobs/7274331253#step:8:59)
  
[59](https://github.com/Southpaw1496/homebrew-quilt/actions/runs/4195113451/jobs/7274331253#step:8:60)
  Caused by:
[60](https://github.com/Southpaw1496/homebrew-quilt/actions/runs/4195113451/jobs/7274331253#step:8:61)
    failed to verify the checksum of `miniz_oxide v0.5.1`
```

I thought it was due to a network error resulting in a corrupted file, but I ran the workflow again and it failed with the same error. I don't know what else could be causing the error or how to fix it. The formula builds correctly on my local Mac, so I think that would indicate something in the Actions pipeline is at fault.

I didn't include `brew config` or `brew doctor` because I don't think either of those apply to GitHub Actions.

[Repository](https://github.com/Southpaw1496/homebrew-quilt)
[Pull Request](https://github.com/Southpaw1496/homebrew-quilt)
[Action Run](https://github.com/Southpaw1496/homebrew-quilt/actions/runs/4195113451/jobs/7274331253)

**Comment by carlocab** [comment]
That's an error coming from `cargo`. I think you'll have to ask the maintainers of `miniz_oxide` about this.

**Reply by Southpaw1496** [reply]
If it was their fault, it wouldn't work in my local environment either, would it?

I imagine they'd just say it was run using a Homebrew command and therefore unsupported, and then I'd be back to square one.

I have found this issue https://github.com/rust-lang/cargo/issues/7180 but I don't understand how it would still work on my local machine.

**Reply by carlocab** [reply]
Well, it doesn't have to be their fault -- they'll just likely know more about what to do about it. If it works for you locally though then it could just be a network failure. Restart your workflow and see if that helps.

**Reply by Southpaw1496** [reply]
I've done that twice, but you make a good point about them knowing what to do.

**Reply by Southpaw1496** [reply]
It's now failed again, but picked a different formula to fail to verify. Maybe GitHub Actions is having a bad day?

Anyway, it's starting to seem like I should ask the Cargo people. Thank you for your help
